build.gradle

```diff
@@ -0,0 +1,137 @@
+// TODO(bdrutu): Default build directory is build but on MacOs with a case-insensitive
+// File System would conflict with Bazel BUILD file.  Remove this and update the
+// .gitignore entry once Bazel is removed.
+buildDir = 'build_gradle'
+
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.4.0'
+        classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.9'
+    }
+}
+
+subprojects {
+    apply plugin: "checkstyle"
+    apply plugin: 'maven'
+    apply plugin: 'idea'
+    apply plugin: 'java'
+    apply plugin: "signing"
+    apply plugin: "jacoco"
+    // TODO(bdrutu): enable all checks.
+    apply plugin: "net.ltgt.errorprone"
+
+    group = "com.google.instrumentation"
+    version = "0.1.0-SNAPSHOT" // CURRENT_INSTRUMENTATION_VERSION
+
+    sourceCompatibility = 1.6
+    targetCompatibility = 1.6
+
+    repositories {
+        mavenCentral()
+        mavenLocal()
+    }
+
+    [compileJava, compileTestJava].each() {
+        it.options.compilerArgs += ["-Xlint:all", "-Xlint:-try", "-Xlint:-path"]
```

Comment:
Was an error in one of the jar in the package com.google.errorprone:error_prone_core:2.0.15. Updated to com.google.errorprone:error_prone_core:2.0.19 works fine now without removing the path warning.